nucypher/policy/collections.py

```diff
@@ -290,15 +291,15 @@ def construct_by_bob(cls, arrangement_id, capsules, ursula, bob):
         if ursula._stamp_has_valid_signature_by_worker():
             ursula_identity_evidence = ursula.decentralized_identity_evidence
 
-        tasks, tasks_bytes = [], []
+        tasks, tasks_bytes = {}, []
```

Comment:
Is it important for `tasks_bytes` to have the same order as tasks? (I understand that in Python 3.7+ it'll be the case anyway)

Reply:
:heavy_check_mark: